<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AwMarkerInspectorUI</class>
 <widget class="QWidget" name="AwMarkerInspectorUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>512</width>
    <height>603</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Adding Markers</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../resources/AnyWave.qrc">
    <normaloff>:/images/AnyWave_icon.png</normaloff>:/images/AnyWave_icon.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_5">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Marker properties</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_8">
        <item>
         <widget class="QRadioButton" name="radioSingle">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Single</string>
          </property>
          <property name="checked">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="radioSelection">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Selection</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,0,0,0,0,0,0,0,0">
        <item>
         <widget class="QLabel" name="labelName">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Label:</string>
          </property>
          <property name="scaledContents">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEditName">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="toolTip">
           <string>Name the marker</string>
          </property>
          <property name="clearButtonEnabled">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="labelValue">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Value:</string>
          </property>
          <property name="scaledContents">
           <bool>true</bool>
          </property>
          <property name="buddy">
           <cstring>spinValue</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="spinValue">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="buttonSymbols">
           <enum>QAbstractSpinBox::NoButtons</enum>
          </property>
          <property name="minimum">
           <double>-1.000000000000000</double>
          </property>
          <property name="maximum">
           <double>1000000000000000013287555072.000000000000000</double>
          </property>
          <property name="value">
           <double>-1.000000000000000</double>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Color:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboBoxColor">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="checkAutoInc">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Auto increment</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <spacer name="horizontalSpacer_5">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_13">
     <item>
      <widget class="QCheckBox" name="checkPredefined">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Use predefined markers</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_11">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBoxPredefined">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Predefined markers</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QTableWidget" name="table">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>150</height>
         </size>
        </property>
        <property name="editTriggers">
         <set>QAbstractItemView::NoEditTriggers</set>
        </property>
        <property name="alternatingRowColors">
         <bool>true</bool>
        </property>
        <property name="selectionMode">
         <enum>QAbstractItemView::MultiSelection</enum>
        </property>
        <property name="selectionBehavior">
         <enum>QAbstractItemView::SelectRows</enum>
        </property>
        <property name="columnCount">
         <number>0</number>
        </property>
        <attribute name="horizontalHeaderVisible">
         <bool>true</bool>
        </attribute>
        <attribute name="horizontalHeaderCascadingSectionResizes">
         <bool>false</bool>
        </attribute>
        <attribute name="horizontalHeaderMinimumSectionSize">
         <number>30</number>
        </attribute>
        <attribute name="horizontalHeaderDefaultSectionSize">
         <number>200</number>
        </attribute>
        <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
         <bool>false</bool>
        </attribute>
        <attribute name="horizontalHeaderStretchLastSection">
         <bool>true</bool>
        </attribute>
        <attribute name="verticalHeaderVisible">
         <bool>false</bool>
        </attribute>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QPushButton" name="buttonAddMarker">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Add Marker</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>13</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="buttonRemoveSelection">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Remove Selection</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="buttonClearList">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Clear</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBoxSettings">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Target type</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QRadioButton" name="radioGlobal">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="toolTip">
           <string>Marker will be global</string>
          </property>
          <property name="text">
           <string>Global marker</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="radioAutoDetect">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="toolTip">
           <string>Mouse click over a channel to make it the target for the marker</string>
          </property>
          <property name="text">
           <string>Auto detect</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="radioSpecific">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="toolTip">
           <string>Marker will target specific channel(s)</string>
          </property>
          <property name="text">
           <string>Channel specifc</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_6">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QListWidget" name="listWidgetTargets">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>150</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <spacer name="horizontalSpacerTargets">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="buttonClearTargets">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="toolTip">
           <string>Clear actual targets</string>
          </property>
          <property name="text">
           <string>Clear</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="buttonAddChannels">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="toolTip">
           <string>Add channel(s) as targets for the marker</string>
          </property>
          <property name="text">
           <string>Add Channels</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../../resources/AnyWave.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>radioGlobal</sender>
   <signal>toggled(bool)</signal>
   <receiver>listWidgetTargets</receiver>
   <slot>setHidden(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>77</x>
     <y>425</y>
    </hint>
    <hint type="destinationlabel">
     <x>66</x>
     <y>456</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioSpecific</sender>
   <signal>toggled(bool)</signal>
   <receiver>listWidgetTargets</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>297</x>
     <y>425</y>
    </hint>
    <hint type="destinationlabel">
     <x>173</x>
     <y>441</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioGlobal</sender>
   <signal>toggled(bool)</signal>
   <receiver>buttonAddChannels</receiver>
   <slot>setHidden(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>107</x>
     <y>425</y>
    </hint>
    <hint type="destinationlabel">
     <x>527</x>
     <y>647</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioSpecific</sender>
   <signal>toggled(bool)</signal>
   <receiver>buttonAddChannels</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>297</x>
     <y>425</y>
    </hint>
    <hint type="destinationlabel">
     <x>527</x>
     <y>647</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkPredefined</sender>
   <signal>toggled(bool)</signal>
   <receiver>groupBoxPredefined</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>146</x>
     <y>116</y>
    </hint>
    <hint type="destinationlabel">
     <x>314</x>
     <y>200</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonAddMarker</sender>
   <signal>clicked()</signal>
   <receiver>AwMarkerInspectorUI</receiver>
   <slot>addPredefinedMarker()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>84</x>
     <y>364</y>
    </hint>
    <hint type="destinationlabel">
     <x>6</x>
     <y>186</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonClearList</sender>
   <signal>clicked()</signal>
   <receiver>AwMarkerInspectorUI</receiver>
   <slot>clearPredefinedMarkers()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>527</x>
     <y>364</y>
    </hint>
    <hint type="destinationlabel">
     <x>315</x>
     <y>218</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonRemoveSelection</sender>
   <signal>clicked()</signal>
   <receiver>AwMarkerInspectorUI</receiver>
   <slot>removeSelectedPredefinedMarkers()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>399</x>
     <y>364</y>
    </hint>
    <hint type="destinationlabel">
     <x>315</x>
     <y>172</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkPredefined</sender>
   <signal>toggled(bool)</signal>
   <receiver>AwMarkerInspectorUI</receiver>
   <slot>changeUsePredefined(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>146</x>
     <y>116</y>
    </hint>
    <hint type="destinationlabel">
     <x>315</x>
     <y>48</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonAddChannels</sender>
   <signal>clicked()</signal>
   <receiver>AwMarkerInspectorUI</receiver>
   <slot>addTargets()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>527</x>
     <y>647</y>
    </hint>
    <hint type="destinationlabel">
     <x>311</x>
     <y>701</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioSpecific</sender>
   <signal>toggled(bool)</signal>
   <receiver>AwMarkerInspectorUI</receiver>
   <slot>changeTargetChannels(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>297</x>
     <y>425</y>
    </hint>
    <hint type="destinationlabel">
     <x>312</x>
     <y>597</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioGlobal</sender>
   <signal>toggled(bool)</signal>
   <receiver>AwMarkerInspectorUI</receiver>
   <slot>changeTargetChannels(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>55</x>
     <y>425</y>
    </hint>
    <hint type="destinationlabel">
     <x>5</x>
     <y>418</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonClearTargets</sender>
   <signal>clicked()</signal>
   <receiver>AwMarkerInspectorUI</receiver>
   <slot>clearTargets()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>356</x>
     <y>647</y>
    </hint>
    <hint type="destinationlabel">
     <x>370</x>
     <y>651</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioAutoDetect</sender>
   <signal>toggled(bool)</signal>
   <receiver>listWidgetTargets</receiver>
   <slot>setHidden(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>193</x>
     <y>425</y>
    </hint>
    <hint type="destinationlabel">
     <x>129</x>
     <y>460</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioAutoDetect</sender>
   <signal>toggled(bool)</signal>
   <receiver>buttonClearTargets</receiver>
   <slot>setHidden(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>193</x>
     <y>425</y>
    </hint>
    <hint type="destinationlabel">
     <x>356</x>
     <y>647</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioGlobal</sender>
   <signal>toggled(bool)</signal>
   <receiver>buttonClearTargets</receiver>
   <slot>setHidden(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>107</x>
     <y>425</y>
    </hint>
    <hint type="destinationlabel">
     <x>356</x>
     <y>647</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioAutoDetect</sender>
   <signal>toggled(bool)</signal>
   <receiver>buttonAddChannels</receiver>
   <slot>setHidden(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>193</x>
     <y>425</y>
    </hint>
    <hint type="destinationlabel">
     <x>527</x>
     <y>647</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioAutoDetect</sender>
   <signal>toggled(bool)</signal>
   <receiver>AwMarkerInspectorUI</receiver>
   <slot>changeTargetChannels(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>193</x>
     <y>425</y>
    </hint>
    <hint type="destinationlabel">
     <x>1</x>
     <y>619</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>addPredefinedMarker()</slot>
  <slot>clearPredefinedMarkers()</slot>
  <slot>removeSelectedPredefinedMarkers()</slot>
  <slot>changeUsePredefined(bool)</slot>
  <slot>addTargets()</slot>
  <slot>changeTargetChannels(bool)</slot>
  <slot>editKeyBindings()</slot>
  <slot>changeUseShortcut(bool)</slot>
  <slot>changeKeyBinding()</slot>
  <slot>changeComboBinding()</slot>
  <slot>clearTargets()</slot>
 </slots>
</ui>
